packages/blockchain-extension/extension/commands/addEnvironmentCommand.ts

```diff
@@ -94,78 +97,153 @@ export async function addEnvironment(): Promise<void> {
 
             envDir = chosenUri.fsPath;
         } else if (createMethod === UserInputUtil.ADD_ENVIRONMENT_FROM_OPS_TOOLS) {
-            const keytar: any = ModuleUtil.getCoreNodeModule('keytar');
-            if (!keytar) {
-                throw new Error('Error importing the keytar module');
-            }
-
-            const HEALTH_CHECK: string = '/ak/api/v1/health';
-            const GET_ALL_COMPONENTS: string = '/ak/api/v1/components';
-            let url: string;
-            const userUrl: string = await UserInputUtil.showInputBox('Enter the URL of the IBM Blockchain Platform Console you want to connect to');
-            if (!userUrl) {
+            const isSaaS: string = await UserInputUtil.showQuickPickYesNo('Are you connecting to a service instance on IBM Cloud?');
+            if (!isSaaS) {
                 return;
+            } else if (isSaaS === UserInputUtil.YES) {
+                let url: string;
```

Comment:
can we split some of this file out into functions. The functions can be in this file but its getting to be a really big function

Reply:
done.